osf/migrations/0177_update_schema_data.py

```diff
@@ -0,0 +1,285 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.13 on 2018-09-26 20:08
+from __future__ import unicode_literals
+
+import re
+import logging
+from django.db import migrations
+from osf.models.base import generate_object_id
+
+logger = logging.getLogger(__file__)
+
+# Dict to map original schema formats to new blocks types
+FORMAT_TYPE_TO_TYPE_MAP = {
+    ('multiselect', 'choose'): 'multi-select-input',
+    (None, 'multiselect'): 'multi-select-input',
+    (None, 'choose'): 'single-select-input',
+    ('osf-upload-open', 'osf-upload'): 'file-input',
+    ('osf-upload-toggle', 'osf-upload'): 'file-input',
+    ('singleselect', 'choose'): 'single-select-input',
+    ('text', 'string'): 'short-text-input',
+    ('textarea', 'osf-author-import'): 'contributors-input',
+    ('textarea', 'string'): 'long-text-input',
+    ('textarea-lg', None): 'long-text-input',
+    ('textarea-lg', 'string'): 'long-text-input',
+    ('textarea-xl', 'string'): 'long-text-input',
+}
+
+def remove_version_1_schemas(state, schema):
+    RegistrationSchema = state.get_model('osf', 'registrationschema')
+    assert RegistrationSchema.objects.filter(schema_version=1, abstractnode__isnull=False).count() == 0
+    assert RegistrationSchema.objects.filter(schema_version=1, draftregistration__isnull=False).count() == 0
+    RegistrationSchema.objects.filter(schema_version=1).delete()
```

Comment:
remove_version_1_schemas already existed as part of Matt's work - just moved to new file.